### PR TITLE
make count limited in all api

### PR DESCRIPTION
**Pull request by xlqian**
http://jira.canaltp.fr/browse/NAVP-508

### CODE REVIEW

File: source/jormungandr/tests/isochrone_tests.py [hunk 1]
```diff
@@ -122,5 +122,6 @@ def test_isochrone_count(self):
         is_valid_isochrone_response(response, self.tester, query)
         query += "&count=1"
         response = self.query(query)
+        print len(response["journeys"])
```
Comment (antoine-de) [inline source/jormungandr/tests/isochrone_tests.py]: debug no ?

File: source/jormungandr/tests/proxy_realtime_tests.py [hunk 2]
```diff
@@ -144,4 +144,5 @@ def test_departures(self):
             ("L", "20160102T100700"),
             ("L", "20160102T101100"),
             ]
+        print departures
```
Comment (antoine-de) [inline source/jormungandr/tests/proxy_realtime_tests.py]: debug no ?

**Comment by xlqian** [comment]
Jenkins,  test this please!

**Comment by antoine-de** [comment]
Ok for me

File: source/jormungandr/tests/isochrone_tests.py [hunk 3]
```diff
@@ -124,3 +124,12 @@ def test_isochrone_count(self):
...
         assert len(response["journeys"]) == 1
         is_valid_isochrone_response(response, self.tester, query)
+
+    def test_invalid_count(self):
+        query = "v1/coverage/basic_routing_test/stop_points/A/journeys?max_duration=25500&datetime=20120614T080000"
+        response = self.query(query)
+        assert len(response["journeys"]) == 2
+        # invalid count
+        query += "&count=toto"
+        response = self.query(query, might_have_additional_args=True)
```
Comment (antoine-de) [inline source/jormungandr/tests/isochrone_tests.py]: the better way to do this is self.query_no_assert I think, it returns the response + an http error code that you might check

**Pull request merged**